k8s/src/main/scala/io/buoyant/k8s/IngressCache.scala

```diff
@@ -83,19 +83,16 @@ class IngressCache(namespace: Option[String], apiClient: Service[Request, Respon
 
   private[this] lazy val ingresses: Activity[Seq[IngressSpec]] = {
     val act = api.activity(unpackIngressList) {
-      (ingresses, watchEvent) =>
-        watchEvent match {
-          case v1beta1.IngressAdded(a) => ingresses ++ mkIngress(a)
-          case v1beta1.IngressModified(m) =>
-            mkIngress(m)
-              .map { item => ingresses.filterNot(isNameEqual(_, item)) :+ item }
-              .getOrElse(ingresses)
-          case v1beta1.IngressDeleted(_) =>
-            Seq.empty
-          case v1beta1.IngressError(e) =>
-            log.error("k8s watch error: %s", e)
-            ingresses
-        }
+      case (oldIngresses, v1beta1.IngressAdded(a)) => oldIngresses ++ mkIngress(a)
```

Comment:
Is this just a cosmetic/style change?

Reply:
Yeah, this class was changed to use the `NewState` extractor, and then it was changed back, but I had restructured the pattern match slightly. Can just revert.